### ISSUE TITLE
Example and consumption of STM32F411CEU6 in deep sleep

### ISSUE THREAD

**Issue by snorkman88** [dukeduck1984/WEACT_STM32F411CEU6#1]
Awesome repo, very well documented!
Have you tested the deep sleep function? Do you have an example?
Have you measured the deep sleep current consumption?

**Comment by dukeduck1984** [comment]
No.  I just did 'hello world' test in REPL, and nothing beyond that point.  Pls let me know if you have done the test :-)
